flow/project.py

```diff
@@ -3401,7 +3887,8 @@ def _register_groups(self):
                 operation_directives = getattr(func, '_flow_group_operation_directives', dict())
                 for group_name in func._flow_groups:
                     self._groups[group_name].add_operation(
-                        op_name, op, operation_directives.get(group_name, None))
+                        op_name, op, operation_directives.get(group_name, None),
+                        flow_aggregate[op_name], flow_select[op_name])
```

Comment:
See comment in `FlowGroup.add_operation`. I don't see a need for `flow_aggregate` or `flow_select` being parameters.